cmd/neutree-cli/app/util/ip.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"fmt"
+	"net"
+)
+
+func GetHostIP() (string, error) {
```

Comment:
maybe we can just let the user pass in this as arguments?

Reply:
use the existing `--deploy-ips `args ? if deploy mode is single, we choose the first ip as the deploy args

Reply:
This also seems to be a bad process, and there may be problems with cluster deployment. I will add the `--node-ip` parameter to solve this problem.